src/jaegertracing/TracerFactoryTest.cpp

```diff
@@ -27,11 +29,11 @@ TEST(TracerFactory, testInvalidConfig)
                                              R"({
       "service_name": {}
     })" };
-    TracerFactory tracerFactory;
+    const auto tracerFactory = new (std::nothrow) TracerFactory(true);
```

Comment:
There is a memory leak here. There is no need to call new. TracerFactory tracerFactory(new) is sufficient.